abs.yaml

```diff
@@ -0,0 +1,10 @@
+# the conda build parameters to use
+build_parameters:
+  - "--suppress-variables"
+  - "--skip-existing"
+  - "--error-overlinking"
+  - "--variants \"{skip_gpu: True}\""
```

Comment:
Nice trick there! We might want to call it `skip_cuda_prefect` though. Right now the package hash contains `"skip_gpu": "True"` for the metal variant.